src/main/java/com/hubspot/jinjava/features/DelegatingFeatureActivationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package com.hubspot.jinjava.features;
+
+import java.util.function.Supplier;
+
+public class DelegatingFeatureActivationStrategy implements FeatureActivationStrategy {
```

Comment:
I think `DisabledFeatureActivationStrategy`, `DelegatingFeatureActivationStrategy`, and `StaticFeatureActivationStrategy` are redundant implementations as you can use a lambda:
```java
FeatureConfig
  .newBuilder()
  .add(name1, () -> false)
  .add(name2, () -> true)
  .add(name3, supplier)
```

Reply:
The `DelegatingFeatureActivationStrategy` boilerplate can be removed too

Reply:
Done and also isActive now gets `Context` objects so it can make decisions based on other data that's in there, perhaps a user or customer userid.